__tests__/app.test.js

```diff
@@ -37,4 +37,40 @@ describe('api/', () => {
             expect(response.body).toEqual(endpointsObj)
         })
     })
+})
+describe('/api/articles/:article_id', () => {
+    test('GET:200 sends a single article to the client', () => {
+      return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body}) => {
+            delete body.msg.created_at
```

Comment:
I see why you have done this, but it is cheating! 😱 The good news is, you don't have to explicitly test the value of ```created_at``` and you can use ```expect.any(String)``` The only explicit value you need to check here is that the ```article_id``` is 1 🙂